ui/components/ui/tabs/tabs.types.ts

```diff
@@ -2,21 +2,15 @@ import React from 'react';
 import { type BoxProps, type TextProps } from '@metamask/design-system-react';
 
 export type TabProps<TKey extends string = string> = Omit<
-  BoxProps,
+  TextProps,
   'children' | 'onClick' | 'ref'
 > & {
-  /** CSS class name for the active tab state */
-  activeClassName?: string;
-  /** CSS class name for the tab button */
-  buttonClassName?: string;
   /** CSS class name for the tab container */
   className?: string;
   /** Test id for the tab element */
   'data-testid'?: string;
   /** Whether the tab is currently active (set by parent Tabs component) */
   isActive?: boolean;
-  /** Whether this is the only tab (set by parent Tabs component) */
-  isSingleTab?: boolean;
```

Comment:
Not used 

<img width="316" height="267" alt="Screenshot 2025-09-18 at 2 17 19 PM" src="https://github.com/user-attachments/assets/450122ae-f036-4244-a5b9-2d59c70107b2" />